glance/server.sls

```diff
@@ -79,6 +79,27 @@ glance_install_database:
   - require:
     - service: glance_services
 
+{%- if server.get('image_cache', {}).get('enabled', False) %}
+glance_cron_glance-cache-pruner:
```

Comment:
The current use-case for our deployment is not to have a shared image-cache yet, so for now the cronjob needs to run on all control nodes.